Command was killed with SIGABRT (Aborted): pkg-pr-new publish --compact --pnpm

Hi there, we are getting this error on most of our pkg.pr.new repo github actions, did something change?

```
Progress: resolved 1, reused 0, downloaded 0, added 0
Packages: +53
+++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 53, reused 2[8](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:9), downloaded 25, added 53, done

<--- Last few GCs --->

[1[9](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:10)48:0x5eb7010]   314616 ms: Scavenge 4013.7 (4131.2) -> 40[10](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:11).3 (4131.2) MB, 13.47 / 0.00 ms  (average mu = 0.696, current mu = 0.773) allocation failure; 
[1948:0x5eb7010]   314643 ms: Scavenge 4015.8 (4131.2) -> 40[11](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:12).3 (4131.2) MB, 9.11 / 0.00 ms  (average mu = 0.696, current mu = 0.773) allocation failure; 
[1948:0x5eb7010]   314687 ms: Scavenge 4017.9 (4131.2) -> 40[12](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:13).4 (4147.2) MB, 15.40 / 0.00 ms  (average mu = 0.696, current mu = 0.773) allocation failure; 


<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
----- Native stack trace -----

 1: 0xb86ecf node::OOMErrorHandler(char const*, v8::OOMDetails const&) [node]
 2: 0xef74d0 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [node]
 3: 0xef77b7 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [node]
 4: 0x1109355  [node]
 5: 0x11211d8 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
 6: 0x10f72f1 v8::internal::HeapAllocator::AllocateRawWithLightRetrySlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
 7: 0x10f8485 v8::internal::HeapAllocator::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
 8: 0x10d4ba6 v8::internal::Factory::AllocateRaw(int, v8::internal::AllocationType, v8::internal::AllocationAlignment) [node]
 9: 0x10c67d4 v8::internal::FactoryBase<v8::internal::Factory>::AllocateRawWithImmortalMap(int, v8::internal::AllocationType, v8::internal::Map, v8::internal::AllocationAlignment) [node]
10: 0x10c8fb6 v8::internal::FactoryBase<v8::internal::Factory>::NewRawOneByteString(int, v8::internal::AllocationType) [node]
11: 0x141a297 v8::internal::String::SlowFlatten(v8::internal::Isolate*, v8::internal::Handle<v8::internal::ConsString>, v8::internal::AllocationType) [node]
12: 0x155334d v8::internal::Runtime_StringCharCodeAt(int, unsigned long*, v8::internal::Isolate*) [node]
[13](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:14): 0x7fef918d9ef6 
 ERROR  Command was killed with SIGABRT (Aborted): pkg-pr-new publish --compact --pnpm

pnpm: Command was killed with SIGABRT (Aborted): pkg-pr-new publish --compact --pnpm
    at makeError (/home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:19342:17)
    at handlePromise (/home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:19913:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.handler [as dlx] (/home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:2196[17](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:18):9)
    at async /home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:229028:[21](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:22)
    at async main (/home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:[22](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:23)8991:34)
    at async runPnpm (/home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:229242:5)
    at async /home/runner/.cache/node/corepack/v1/pnpm/9.1.4/dist/pnpm.cjs:229[23](https://github.com/Tresjs/tres/actions/runs/11113957016/job/30879427296?pr=845#step:7:24)4:7
Error: Process completed with exit code 1.
```

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -90,7 +90,7 @@ const main = defineCommand({
           },
         },
         run: async ({ args }) => {
-          const paths = args._.length > 0
+          const paths = args._.length
             ? await glob(args._, {
                 expandDirectories: false,
                 onlyDirectories: true,
@@ -98,11 +98,27 @@ const main = defineCommand({
               })
             : [process.cwd()];
 
-          const templates = await glob(args.template ?? [], {
+          if (args._.includes(".") || args._.includes("./")) {
+            paths.push(process.cwd());
+          }
+
+          const templatePatterns =
+            typeof args.template === "string"
+              ? [args.template]
+              : ([...(args.template || [])] as string[]);
+
+          const templates = await glob(templatePatterns, {
             expandDirectories: false,
             onlyDirectories: true,
             absolute: true,
           });
+
+          if (
+            templatePatterns.includes(".") ||
+            templatePatterns.includes("./")
+          ) {
+            templates.push(process.cwd());
+          }
 
           const formData = new FormData();
 

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -91,12 +91,13 @@ const main = defineCommand({
         },
         run: async ({ args }) => {
           console.log('args', args)
+          const ignoredPaths = ["**/node_modules/**", ".git"]
           const paths = args._.length > 0
             ? await glob(args._, {
                 expandDirectories: false,
                 onlyDirectories: true,
                 absolute: true,
-                ignore: ["**/node_modules", ".git"],
+                ignore: ignoredPaths,
               })
             : [process.cwd()];
           console.log('paths', paths)
@@ -105,7 +106,7 @@ const main = defineCommand({
             expandDirectories: false,
             onlyDirectories: true,
             absolute: true,
-            ignore: ["**/node_modules", ".git"],
+            ignore: ignoredPaths,
           });
 
           const formData = new FormData();
@@ -243,7 +244,7 @@ const main = defineCommand({
               cwd: templateDir,
               dot: true,
               onlyFiles: true,
-              ignore: ["**/node_modules", ".git"], // always ignore node_modules and .git
+              ignore: ignoredPaths, // always ignore node_modules and .git
             });
 
             const filteredFiles = files.filter((file) => !ig.ignores(file));

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -90,23 +90,18 @@ const main = defineCommand({
           },
         },
         run: async ({ args }) => {
-          console.log('args', args)
-          const ignoredPaths = ["**/node_modules/**", ".git"]
           const paths = args._.length > 0
             ? await glob(args._, {
                 expandDirectories: false,
                 onlyDirectories: true,
                 absolute: true,
-                ignore: ignoredPaths,
               })
             : [process.cwd()];
-          console.log('paths', paths)
 
           const templates = await glob(args.template ?? [], {
             expandDirectories: false,
             onlyDirectories: true,
             absolute: true,
-            ignore: ignoredPaths,
           });
 
           const formData = new FormData();
@@ -244,7 +239,7 @@ const main = defineCommand({
               cwd: templateDir,
               dot: true,
               onlyFiles: true,
-              ignore: ignoredPaths, // always ignore node_modules and .git
+              ignore: ["**/node_modules", ".git"], // always ignore node_modules and .git
             });
 
             const filteredFiles = files.filter((file) => !ig.ignores(file));

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -90,6 +90,7 @@ const main = defineCommand({
           },
         },
         run: async ({ args }) => {
+          console.log('args', args)
           const paths = args._.length > 0
             ? await glob(args._, {
                 expandDirectories: false,
@@ -97,6 +98,7 @@ const main = defineCommand({
                 absolute: true,
               })
             : [process.cwd()];
+          console.log('paths', paths)
 
           const templates = await glob(args.template ?? [], {
             expandDirectories: false,

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -91,7 +91,6 @@ const main = defineCommand({
         },
         run: async ({ args }) => {
           console.log('args', args)
-          const ignore = ['**/node_modules/**', '.git']
           const paths = args._.length > 0
             ? await glob(args._, {
                 expandDirectories: false,

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -91,11 +91,13 @@ const main = defineCommand({
         },
         run: async ({ args }) => {
           console.log('args', args)
+          const ignore = ['**/node_modules/**', '.git']
           const paths = args._.length > 0
             ? await glob(args._, {
                 expandDirectories: false,
                 onlyDirectories: true,
                 absolute: true,
+                ignore,
               })
             : [process.cwd()];
           console.log('paths', paths)
@@ -104,6 +106,7 @@ const main = defineCommand({
             expandDirectories: false,
             onlyDirectories: true,
             absolute: true,
+            ignore,
           });
 
           const formData = new FormData();

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -97,7 +97,7 @@ const main = defineCommand({
                 expandDirectories: false,
                 onlyDirectories: true,
                 absolute: true,
-                ignore,
+                ignore: ["**/node_modules", ".git"],
               })
             : [process.cwd()];
           console.log('paths', paths)
@@ -106,7 +106,7 @@ const main = defineCommand({
             expandDirectories: false,
             onlyDirectories: true,
             absolute: true,
-            ignore,
+            ignore: ["**/node_modules", ".git"],
           });
 
           const formData = new FormData();

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -28,7 +28,7 @@
     "package-manager-detector": "^0.1.2",
     "pkg-types": "^1.1.1",
     "query-registry": "^3.0.1",
-    "tinyglobby": "0.2.7"
+    "tinyglobby": "0.2.6"
   },
   "devDependencies": {
     "@pkg-pr-new/utils": "workspace:^",

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -28,7 +28,7 @@
     "package-manager-detector": "^0.1.2",
     "pkg-types": "^1.1.1",
     "query-registry": "^3.0.1",
-    "tinyglobby": "^0.2.7"
+    "tinyglobby": "0.2.7"
   },
   "devDependencies": {
     "@pkg-pr-new/utils": "workspace:^",

